Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -0,0 +1,702 @@
+	Hud_AddEventHandler( pilotEditButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "EditPilotLoadoutsMenu" ) ) )
+	file.loadoutButtons.append( pilotEditButton )
+
+	headerIndex++
+	buttonIndex = 0
+	var titanHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_TITAN" )
+	file.titanHeader = titanHeader
+	file.titanHeaderIndex = headerIndex
+	file.loadoutHeaders.append( titanHeader )
+	var titanSelectButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#SELECT" )
+	file.titanSelectButton = titanSelectButton
+	file.loadoutButtons.append( titanSelectButton )
+	Hud_AddEventHandler( titanSelectButton, UIE_CLICK, TitanSelectButtonHandler )
+	var titanEditButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#EDIT" )
+	Hud_AddEventHandler( titanEditButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "EditTitanLoadoutsMenu" ) ) )
+	file.titanEditButton = titanEditButton
+	file.loadoutButtons.append( titanEditButton )
+
+	headerIndex++
+	buttonIndex = 0
+	var gameHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_GAME" )
+	var leaveButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#LEAVE_MATCH" )
+	Hud_AddEventHandler( leaveButton, UIE_CLICK, OnLeaveButton_Activate )
+	#if DEV
+		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Dev" )
+		Hud_AddEventHandler( devButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "DevMenu" ) ) )
+	#endif
+
+	headerIndex++
+	buttonIndex = 0
+	var dummyHeader = AddComboButtonHeader( comboStruct, headerIndex, "" )
+	var dummyButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "" )
+	Hud_SetVisible( dummyHeader, false )
+	Hud_SetVisible( dummyButton, false )
+
+	headerIndex++
+	buttonIndex = 0
+	file.settingsHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_SETTINGS" )
+	var controlsButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#CONTROLS" )
+	Hud_AddEventHandler( controlsButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "ControlsMenu" ) ) )
+	#if CONSOLE_PROG
+		var avButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#AUDIO_VIDEO" )
+		Hud_AddEventHandler( avButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "AudioVideoMenu" ) ) )
+	#elseif PC_PROG
+		var videoButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#AUDIO" )
+		Hud_AddEventHandler( videoButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "AudioMenu" ) ) )
+		var soundButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#VIDEO" )
+		Hud_AddEventHandler( soundButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "VideoMenu" ) ) )
+	#endif
+
+	// MOD SETTINGS
+	var modSettingsButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Mod Settings" )
+	Hud_AddEventHandler( modSettingsButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "ModSettings" ) ) )
+
+	// sorry, no one reads this...
```

Comment:
Change to something more descriptive, e.g.

> `// Replace FAQ button with modsettings one`

(I assume that's what it does?)

Reply:
addressed in 2ed775523a3e23df4dfebbebf4293c3b9e01fd9a